apps/omg_watcher/config/config.exs

```diff
@@ -5,6 +5,8 @@ config :omg_watcher, child_chain_url: "http://localhost:9656"
 config :omg_watcher,
   # 23 hours worth of blocks - this is how long the child chain server has to block spends from exiting utxos
   exit_processor_sla_margin: 23 * 60 * 4,
+  # this means we don't want the `sla_margin` above be larger than the `min_exit_period`
+  exit_processor_sla_margin_force: false,
```

Comment:
I think `force` -> `forced` is more natural + stops me from chuckling the StarWars pun :trollface: